spec/std/time/time_spec.cr

```diff
@@ -138,6 +139,16 @@ describe Time do
     t1.second.should eq(13)
   end
 
+  pending "add days over dst" do
+    with_zoneinfo do
+      location = Time::Location.load("Europe/Berlin")
+      reference = Time.new(2017, 10, 28, 13, 37, location: location)
+      next_day = Time.new(2017, 10, 28, 13, 37, location: location)
```

Comment:
Yeah, that's a typo. This example is marked as `pending`, thus it's not executed at all (and the error got unnoticed until now). With the current implemenation of `Time::Span` it won't work, because `1.day` equals `24 hours`, but from `2017-10-28 13:37` (CEST +02:00) to `2017-10-29 13:37` (CET +01:00) it is only `23 hours` (but still `1 day`).